beta/nncf/tensorflow/pruning/base_algorithm.py

```diff
@@ -36,6 +36,7 @@
 from beta.nncf.tensorflow.pruning.utils import get_filters_num
 from beta.nncf.tensorflow.pruning.utils import is_shared
 from beta.nncf.tensorflow.sparsity.magnitude.operation import BinaryMask
+from beta.nncf.tensorflow.sparsity.magnitude.operation import OP_NAME_BM
```

Comment:
Could you consider using the class method instead of a global variable? I mean `BinaryMask.create_operation_name(layer_name, weight_attr_name)`

Reply:
I agree it would be better, I'll refactor it

Reply:
Done